langchain4j-google-ai-gemini/src/test/java/dev/langchain4j/model/googleai/GoogleAiGeminiChatModelIT.java

```diff
@@ -674,6 +674,35 @@ void should_support_tool_config() {
         assertThat(chatResponse.aiMessage().hasToolExecutionRequests()).isFalse();
     }
 
+    @Test
+    void should_count_tokens() {
+        // given
+        GoogleAiGeminiChatModel gemini = GoogleAiGeminiChatModel.builder()
+            .apiKey(GOOGLE_AI_GEMINI_API_KEY)
+            .modelName("gemini-1.5-flash")
+            .logRequestsAndResponses(true)
+            .build();
+
+        // when
+        int countedTokens = gemini.estimateTokenCount("What is the capital of France?");
+
+        // then
+        assertThat(countedTokens).isGreaterThan(0);
```

Comment:
It would probably be nice to assert the real number of tokens in these tests. Otherwise we could break something and we won't notice it.